deps/src/main/java/com/microsoft/azure/sdk/iot/deps/convention/GsonPayloadSerializer.java

```diff
@@ -0,0 +1,75 @@
+package com.microsoft.azure.sdk.iot.deps.convention;
+
+import lombok.Getter;
+
+/**
+ * A {@link com.google.gson.Gson} {@link PayloadSerializer} implementation.
+ */
+public class GsonPayloadSerializer extends PayloadSerializer
+{
+    /**
+     * The content type string.
+     */
+    private static final String ApplicationJson = "application/json";
+
+    /**
+     * The default instance of this class.
+     */
+    @Getter
+    static final GsonPayloadSerializer Instance = new GsonPayloadSerializer();
+
+    @Getter
+    String ContentType = ApplicationJson;
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String serializeToString(Object objectToSerialize)
+    {
+        return "";
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public <T> T deserializeToType(String stringToDeserialize)
+    {
+        return (T) (new Object());
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public <T> T convertFromObject(Object objectToConvert)
+    {
+        return (T) (new Object());
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public <T> T getNestedObjectValue(Object nestedObject, String propertyName)
+    {
+        return (T) (new Object());
```

Comment:
Why does this method not use any of its parameters?

Reply:
These files were out of sync from my main dev branch .